src/RA_Core.cpp

```diff
@@ -525,18 +525,18 @@ API BOOL CCONV _RA_OfferNewRAUpdate(const char* sNewVer)
 
 		//_WriteBufferToFile( "BatchUpdater.bat", sBatchUpdater, strlen( sBatchUpdater ) );
 
-		//ShellExecute( NULL,
+		//ShellExecute( nullptr,
 		//	"open",
 		//	"BatchUpdater.bat",
-		//	NULL,
-		//	NULL,
+		//	nullptr,
+		//	nullptr,
 		//	SW_SHOWNORMAL ); 
 
-		ShellExecute(NULL,
+		ShellExecute(nullptr,
 			TEXT("open"),
 			TEXT("http://www.retroachievements.org/download.php"),
-			NULL,
-			NULL,
+			nullptr,
+			nullptr,
```

Comment:
Is it safe to use `nullptr` in the WINAPI calls?

Reply:
Yes, "far* LP" or "near* P" stuff are pointers. I did notice some NULLs in the AppendMenu were actually supposed to be integers. Windows NT is compatible with C++ if you know how to use it, Microsoft themselves recommend using nullptr.